Integrations/ARIA/aria_packet_intelligence.py

```diff
@@ -0,0 +1,1621 @@
+            trigger_value: The threshold that must be met before the alert is triggered.
+            label_sia_group: The name of the group to which the SIA belongs.
+            label_sia_name: The name of the SIA.
+            label_sia_region: The name of the region to which the SIA belongs.
+
+        Returns: Dictionary context data contains useful response information.
+
+        """
+        rule = f'{self._process_ip_address(src_ip)}: '
+
+        rule += self._build_alert_instruction(transport_type, int(tti_index), int(aio_index),
+                                              trigger_type, int(trigger_value)) + ', END'
+
+        data = self._generate_named_rule_data(rule_name, 'src-subnet', rule, 'add', None,
+                                              label_sia_group, label_sia_name, label_sia_region)
+        return self._do_request(data, rule_name, rule)
+
+    def mute_alert_src_subnet(self, rule_name: str, label_sia_group: str = None,
+                              label_sia_name: str = None, label_sia_region: str = None) -> dict:
+        """ Removes a named rule from the source subnet logic block, disabling the alerts.
+
+        Args:
+            rule_name: The name of the rule to delete.
+            label_sia_group: The name of the group to which the SIA belongs.
+            label_sia_name: The name of the SIA.
+            label_sia_region: The name of the region to which the SIA belongs.
+
+        Returns: Dictionary context data contains useful response information.
+
+        """
+        return self._remove_rule(rule_name, 'src-subnet', None, label_sia_group, label_sia_name, label_sia_region)
+
+
+''' HELPER FUNCTIONS '''
+
+
+def func_call(instance: ARIA, func_name: str, command_name: str, demisto_arguments: list, args: dict):
+    """ Helper function used to call different demisto command
+
+    Args:
+        instance: An ARIA instance.
+        func_name: Name of the functions in the ARIA class.
+        command_name: Related demisto command name.
+        demisto_arguments: List of arguments name in the right order.
+        args: Input of demisto arguments dict.
+
+    """
+    arguments_value = []
+    for arg in demisto_arguments:
+        value = args.get(arg)  # get values from demisto command
+        arguments_value.append(value)
+
+    context_entry = getattr(instance, func_name)(*tuple(arguments_value))  # get returned tuple
+
+    table_header = ['Rule', 'Status', 'Endpoints']
+
+    context_name = func_name.title().replace('_', '')
+
+    ec = {
+        f'Aria.{context_name}(val.name && val.name == obj.name)': context_entry
+    }
+
+    readable_output = tableToMarkdown(command_name, context_entry, table_header)
+
+    return readable_output, ec
+
+
+''' COMMAND FUNCTION '''
+
+
+def block_conversation_command(instance, args):
+    demisto_arguments = ['src_ip', 'target_ip', 'rule_name', 'src_port', 'target_port', 'protocol', 'label_sia_group',
+                         'label_sia_name'
```

Comment:
you can just call `handle_proxy()` func instead, it handles it for you

no need to init `PROXY` either

Reply:
Remove PROXY and use handle_proxy() instead